core-navigation/build.gradle.kts

```diff
@@ -1,9 +1,13 @@
 @file:Suppress("UnstableApiUsage")
 plugins {
     id("com.streamplayer.android-library")
-    id("com.streamplayer.compose")
+    id("kotlin-android")
```

Comment:
I think you don't need to apply `kotlin-android`, considering the `com.streamplayer.android-library` is already doing that.